src/modules/sdlog2/sdlog2_messages.h

```diff
@@ -480,7 +481,7 @@ static const struct log_format_s log_formats[] = {
 	LOG_FORMAT(GPS, "QBffLLfffffBHHH",	"GPSTime,Fix,EPH,EPV,Lat,Lon,Alt,VelN,VelE,VelD,Cog,nSat,SNR,N,J"),
 	LOG_FORMAT_S(ATTC, ATTC, "ffff",		"Roll,Pitch,Yaw,Thrust"),
 	LOG_FORMAT_S(ATC1, ATTC, "ffff",		"Roll,Pitch,Yaw,Thrust"),
-	LOG_FORMAT(STAT, "BBBfBB",		"MainState,ArmState,FailsafeState,BatRem,BatWarn,Landed"),
+	LOG_FORMAT(STAT, "BBBfBBf",		"MainS,ArmS,Failsafe,BatRem,BatWarn,Landed,Load"),
```

Comment:
How necessary is this change? What are the gains?

The change of MainState to MainS causes issues with LogMuncher (chart.oznet.ch) requiring changes to the pymavlink library and it would have to support both new and old versions if this stays. The other fields are not really an issue.